tests/utils_tests/test_lazyobject.py

```diff
@@ -284,3 +335,17 @@ def test_list_set(self):
         self.assertNotIn(6, lazy_set)
         self.assertEqual(len(lazy_list), 5)
         self.assertEqual(len(lazy_set), 4)
+
+    # Regression test for #25389
```

Comment:
If you can describe the issue instead of simply "Regression test for #25389" that saves looking up the ticket to figure out the issue.